frontend/javascripts/libs/user_local_storage.js

```diff
@@ -4,7 +4,7 @@ import Store from "oxalis/store";
 
 function prefixKey(key) {
   const { activeUser } = Store.getState();
-  const prefix = !activeUser ? "Anonymous" : activeUser.email;
+  const prefix = !activeUser ? "Anonymous" : `${activeUser.email}-${activeUser.organization}`;
```

Comment:
I did a quick check what functionality this would affect as it basically "removes" all items stored in a user's local storage. Most things are not a big deal (dataset cache, last finished task, last search tags). However, the tracing view layouts are also saved in local storage (`layout_persistence.js`) and it could be annoying to users if everything is reset. It might make sense to add a backward compatibility bridge for the `prefixKey` in `getItem` (where the `activeUser.email` prefix is tried if the new prefix didn't turn up a result)  for a while, which can then later be removed. This should be opt-in as it shouldn't happen for the dataset cache, for example. What do you think?

Reply:
You are right, it would be nice to keep this backwards compatible. I think, one could even argue that the layouts should be stored across the user's organizations. So, getItem/setItem could receive an `isOrganizationSpecific`-setting which is true by default and set to false for the layout stuff. That way, this wouldn't even be a compatibility bridge. What do you think? :)

Reply:
That's even more elegant, great idea :+1:

Reply:
Done!